plugins/devops/python-tools/skills/claude-agent-sdk/references/sessions.md

```
@@ -96,12 +103,14 @@ options: {    options: {
   for await (const message of response) {
     console.log(message)
   }
+
   ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)